code/modules/mob/hear_say.dm

```diff
@@ -167,7 +167,22 @@
 
 /mob/proc/hear_sleep(var/message)
 	var/heard = ""
+	var/opentag = 0
+	var/closetag = 0
 	if(prob(15))
+		while(1) //Remove the HTML formatting so people don't hear colour
```

Comment:
I know, I just couldn't find them because the wiki page is out of date and the GitHub search is case sensitive. Also /proc/strip_html_properly would be the correct usage but it's a bit too strict for this case.